### ISSUE TITLE
[ReadtheDocs] Possible Improvements on Nano Section

### ISSUE THREAD

**Issue by Oscilloscope98** [intel-analytics/BigDL#5338]
#### 1.1 [BigDL-Nano TensorFlow Inference Overview](https://bigdl.readthedocs.io/en/latest/doc/Nano/QuickStart/pytorch_inference.html)

For this page, most of contents seems to be directly copied from "BigDL-Nano PyTorch Inference Overview" part. 
However, Nano PyTorch currently have runtime acceleration enabled, which is not yet realized for Nano TensorFlow.
Directly copying some sentences including the concept of runtime acceleration from PyTorch Inference to TensorFlow Inference could make readers confused.
Such sentences include (but not limited to):

> Currently, performance accelerations are achieved by **integrating extra runtimes as inference backend engines** or using quantization methods on full-precision trained models to reduce computation during inference.

> To use INC as your quantization engine, you can choose accelerator as None or 'onnxruntime'. Otherwise, accelerator='openvino' means using OpenVINO POT to do quantization.

- [ ] It would be better if we could revise this section to introduce runtime accelerator in a reasonable way, e.g. as a up-coming feature, as a convention with PyTorch to have
the `accelerator` parameter for `quantize` function, etc.

#### 2.1 [Nano User Guide](https://bigdl.readthedocs.io/en/latest/doc/Nano/Overview/nano.html)

- [ ] [Install](https://bigdl.readthedocs.io/en/latest/doc/Nano/Overview/nano.html#install): it seems that there should be `conda create -n env python=3.7` instead of `conda create -n env` for creating environment
![image](https://user-images.githubusercontent.com/54161268/183320767-917d4474-044c-4819-ac24-0c9e08697d22.png)

#### 2.2 [Windows User Guide](https://bigdl.readthedocs.io/en/latest/doc/Nano/Overview/windows_guide.html)

- [ ] [Install WSL2](https://bigdl.readthedocs.io/en/latest/doc/Nano/Overview/windows_guide.html#step-1-install-wsl2): It is better to have a link of "WSL manual installation steps" for users with older build
![image](https://user-images.githubusercontent.com/54161268/183321020-14389fc1-130c-467c-b2dd-025e0bd92839.png)

- [ ] [Create a BigDL-Nano env](https://bigdl.readthedocs.io/en/latest/doc/Nano/Overview/windows_guide.html#step-3-create-a-bigdl-nano-env): same problem as before, we should have `conda create -n env python=3.7` instead of `conda create -n env`
![image](https://user-images.githubusercontent.com/54161268/183321046-3ed26e88-d8e4-44a9-8d77-78b2c08981b7.png)

#### 2.3 [BigDL-Nano PyTorch Training Overview](https://bigdl.readthedocs.io/en/latest/doc/Nano/QuickStart/pytorch_train.html)
- [ ] [Best Known Configurations](https://bigdl.readthedocs.io/en/latest/doc/Nano/QuickStart/pytorch_train.html#best-known-configurations): It is not clear here whether the environment variables are set by `source bigdl-nano-init` directly, or this command just generates them and users need to set by themselves. Could we make it more clear?
![image](https://user-images.githubusercontent.com/54161268/183321428-87ffcef1-b9d3-43f6-9b53-8a03f2c26171.png)

- [ ] [BigDL-Nano PyTorch Trainer](https://bigdl.readthedocs.io/en/latest/doc/Nano/QuickStart/pytorch_train.html#bigdl-nano-pytorch-trainer): the code: `from bigdl.nano.pytorch import Trainer` seems to be duplicated here.
![image](https://user-images.githubusercontent.com/54161268/183321575-dafad0c7-10c8-4c2f-9421-7b1480b1ef20.png)

- [ ] [Multi-instance Training](https://bigdl.readthedocs.io/en/latest/doc/Nano/QuickStart/pytorch_train.html#multi-instance-training): it is a little bit abrupt to mention "PyTorch’s DDP API" here. The logic could be clearer if we add a sentence like: Although it is common to use PyTorch Distributed Data Parallel (DDP) API,  it is a little cumbersome and error-prone...
![image](https://user-images.githubusercontent.com/54161268/183322172-c2e96553-2a19-43df-be6e-f1b4a2b9f37e.png)

#### 2.4 [BigDL-Nano PyTorch Inference Overview](https://bigdl.readthedocs.io/en/latest/doc/Nano/QuickStart/pytorch_inference.html#bigdl-nano-pytorch-inference-overview)
- [ ] [BigDL-Nano PyTorch Inference Overview](https://bigdl.readthedocs.io/en/latest/doc/Nano/QuickStart/pytorch_inference.html#bigdl-nano-pytorch-inference-overview): It would be better here to give a brief introduction about ONNXRuntime and OpenVINO, or at least include the links here. 
![image](https://user-images.githubusercontent.com/54161268/183329806-757615db-415b-4ebb-9b91-fb998f138ea9.png)

- [ ] [Quantization](https://bigdl.readthedocs.io/en/latest/doc/Nano/QuickStart/pytorch_inference.html#quantization): "Intel Neural Compressor (INC) and Post-training Optimization Tools (POT) from OpenVINO toolkit are enabled as options." <= in this sentence, it seems like INC is also from OpenVINO toolkit. It would be better to rearrange this sentence. And it could be clearer to add link for INC and POT.
![image](https://user-images.githubusercontent.com/54161268/183330042-72b0cbac-9b55-472e-9c05-4015158ba8f8.png)

- [ ] [Quantization](https://bigdl.readthedocs.io/en/latest/doc/Nano/QuickStart/pytorch_inference.html#quantization): "In the meantime, runtime acceleration is also included directly in the quantization pipeline when using accelerator='onnxruntime'/'openvino' so you don’t have to run Trainer.trace before quantization." => the logic here is a little bit strange. Would it be better to add something like "If you would like to use runtime acceleration together with quantization, runtime acceleration is included directly in the quantization pipeline...”
![image](https://user-images.githubusercontent.com/54161268/183330042-72b0cbac-9b55-472e-9c05-4015158ba8f8.png)

- [ ] [Quantization using Intel Neural Compressor](https://bigdl.readthedocs.io/en/latest/doc/Nano/QuickStart/pytorch_inference.html#quantization-using-intel-neural-compressor): it would be more user-friendly to directly add the installation guide (`pip install onnx onnxruntime`) for ONNXRuntime Acceleration here, instead of redirecting to another place.
![image](https://user-images.githubusercontent.com/54161268/183331978-5c52e357-c2c6-4a39-98c6-8ae27760a167.png)

- [ ] [Quantization with Accuracy Control](https://bigdl.readthedocs.io/en/latest/doc/Nano/QuickStart/pytorch_inference.html#quantization-with-accuracy-control): seems like parameters `precision`, `approach`, and `method`, which are used in the example code below, are not explained here. Is there a specific reason?
![image](https://user-images.githubusercontent.com/54161268/183340413-b354143f-51dc-428a-8c47-b92ebd1f924c.png)

#### 2.5 [BigDL-Nano TensorFlow Training Overview](https://bigdl.readthedocs.io/en/latest/doc/Nano/QuickStart/tensorflow_train.html#bigdl-nano-tensorflow-training-overview)

- [ ] [BigDL-Nano TensorFlow Training Overview](https://bigdl.readthedocs.io/en/latest/doc/Nano/QuickStart/tensorflow_train.html#bigdl-nano-tensorflow-training-overview): Do we have the link know?
![image](https://user-images.githubusercontent.com/54161268/183340756-f5abbd9d-3595-49e1-a52f-81ccce2834d9.png)

- [ ] [Best Known Configurations](https://bigdl.readthedocs.io/en/latest/doc/Nano/QuickStart/tensorflow_train.html#best-known-configurations): Would it be better here to give a brief introduction/link to `intel-tensorflow` or intel’s oneDNN?
![image](https://user-images.githubusercontent.com/54161268/183341083-7b2ce37c-f643-4ac2-aaca-af41a0f3483b.png)

- [ ] [Best Known Configurations](https://bigdl.readthedocs.io/en/latest/doc/Nano/QuickStart/tensorflow_train.html#best-known-configurations): the same problem as the one in [BigDL-Nano PyTorch Training Overview](https://bigdl.readthedocs.io/en/latest/doc/Nano/QuickStart/pytorch_train.html#best-known-configurations); It is not clear here whether the environment variables are set by `source bigdl-nano-init` directly, or this command just generates them and users need to set by themselves.
![image](https://user-images.githubusercontent.com/54161268/183341083-7b2ce37c-f643-4ac2-aaca-af41a0f3483b.png)

### 3. General Issues

- [ ] Not all the subsection titles are numbered. For example, [Nano User Guide](https://bigdl.readthedocs.io/en/latest/doc/Nano/Overview/nano.html) is numbered, 
while [BigDL-Nano PyTorch Training Overview](https://bigdl.readthedocs.io/en/latest/doc/Nano/QuickStart/pytorch_train.html) is not numbered. It is better to be consistent. 
- [ ] There is no hierarchy navigation (or navigation back) for [Nano tutorials](https://bigdl.readthedocs.io/en/latest/doc/Nano/QuickStart/index.html). In any of the Nano tutorials, readers can only navigate back to the home/landing page, instead of the tutorial index page.
- [ ] For tutorials, it would be better to add some assumption/knowledge prerequisite for being able to follow them, so that readers could have a better idea of what they should expect from these tutorials. Such as what PyTorch Tutorials do here:
![image](https://user-images.githubusercontent.com/54161268/183342418-453a2b0b-51ae-45ca-afae-90382c42b310.png)
